### ISSUE TITLE
Create issue for docs?

### ISSUE THREAD

**Issue by travis5491811** [reactioncommerce/reaction#6264]
Hello,

Can you create a issue category for reporting issues with documentation? Right now you have Core, Bugs, and Questions (that will be immediately closed so no point in posting there) and Gitter. Lots of stuff gets lost in "Gitter" and because all conversations are mixed together in a single thread, its hard to follow (where did conversation start? when did question get resolved? who is x responding to?). That's a separate concern. I'm primarily here about the docs.

Did I miss something? I'm currently evaluating reaction to see if I will introduce to our team. When going through the docs, [this page](https://docs.reactioncommerce.com/docs/installation-reaction-platform) appears incorrect. 

> 7. Congrats 🎉 Now you're running the entire suite of Reaction Development Platform services and ready to start developing.
> **Note: To login into the Operator UI use the credentials found in the [env.example](https://github.com/reactioncommerce/reaction/blob/v2.9.1/.env.example#L11-L12) file.**

When I try that login, its invalid. My .env.example file does not have the following two lines:

```
REACTION_AUTH=r3@cti0n
REACTION_EMAIL=admin@localhost
```

This is what my `reaction/.env.example` looks like:

```
MONGO_URL=mongodb://mongo.reaction.localhost:27017/reaction
ROOT_URL=http://localhost:3000
STRIPE_API_KEY=YOUR_PRIVATE_STRIPE_API_KEY
```

Plus my mongodb users collection was empty.  In order to access `http://localhost:4080/` I had to create a user through the front-end GUI

[This page](https://github.com/reactioncommerce/reaction) says to get started follow [this page](https://docs.reactioncommerce.com/docs/installation-reaction-platform) but after following that page and getting everything running correctly based on that page if I go back to continue on the previous page to "Start App in Docker Container (Recommended)", those steps are already done (mondodb container and reaction API already running).

I've managed to mussel my way into playing with reaction. But as a person new to the project, it's confusing trying to follow what I should be doing next when the initial goal is to get a demo up and running and see how well it works. I'm not sure if there is anything that can be done about it other than mentioning it. Maybe I'm the only one and/or I missed a step.

Currently evaluating: 

```
API Version: 3.8.0
MongoDB Version: 4.2.0
Installed Plugins:
simple-schema 1.0.0
job-queue 1.0.1
files 1.0.0
shops 1.0.1
settings 1.0.0
i18n 1.0.0
email 1.0.0
address-validation 1.3.0
plugin-translations 1.1.0
system-information 1.1.0
email-smtp 1.0.0
email-templates 1.0.0
accounts 1.3.0
authentication 1.1.0
authorization-simple 1.2.0
products 1.0.0
catalogs 1.0.2
tags 1.0.0
pricing-simple 1.0.0
inventory 1.0.1
inventory-simple 1.0.0
carts 1.2.0
orders 1.2.0
payments 1.0.0
payments-stripe 1.0.0
payments-example 1.1.0
discounts 1.0.0
discounts-codes 3.8.0
surcharges 1.1.0
shipments 1.0.0
shipments-flat-rate 1.0.0
taxes 1.0.0
taxes-flat-rate 1.0.1
navigation 1.1.0
sitemap-generator 1.2.0
notifications 1.0.0
address-validation-test 1.0.0
```

**Comment by focusaurus** [comment]
For docs bugs, you can file them as issues in the [reaction-docs](https://github.com/reactioncommerce/reaction-docs) github repo.

**Comment by ghost** [comment]
@travis5491811 I wasn't able to find your issue in the docs repo. To get your demo running perhaps you may find the following tutorial helpful in the future: https://habd.as/post/reaction-commerce-getting-started/
